tensorflow/stream_executor/blas.h

```diff
@@ -1383,6 +1455,93 @@ class BlasSupport {
                           const DeviceMemory<std::complex<double>> &a, int lda,
                           DeviceMemory<std::complex<double>> *b, int ldb) = 0;
 
+  // Creates a backend-specific plan object for a blaslt matmul operation, which
+  // can then be passed to DoBlasLtMatmul(). When possible, plans should be
+  // created once and reused for multiple calls to DoBlasLtMatmul().
+  // Returns a null pointer on failure.
+  std::unique_ptr<blas::IBlasLtMatmulPlan> CreateBlasLtMatmulPlan(
+      blas::DataType ab_type, blas::DataType c_type,
+      blas::ComputationType computation_type, blas::PointerMode pointer_mode,
+      blas::Transpose transa, blas::Transpose transb, uint64 m, uint64 n,
+      uint64 k, int64 lda, int64 ldb, int64 ldc) {
+    return CreateBlasLtMatmulPlanStridedBatched(
+        ab_type, c_type, computation_type, pointer_mode, transa, transb, m, n,
+        k, 1, lda, 0, ldb, 0, ldc, 0);
```

Comment:
Would the overloading still be necessary if we have `MatmulPlanParams` above with defaulted strides?

Reply:
Removed